actionRuntimes/actionProxy/invoke.py

```diff
@@ -3,9 +3,9 @@
 
   This script is useful for testing the action proxy (or its derivatives)
   by simulating invoker interactions. Use it in combination with
-  delete-build-run.sh which builds and starts up the action proxy.
-  Examples:
-     ./delete-build-run.sh &
+  docker run <image> which starts up the action proxy.
+  Example:
+     docker run -i -t -p 8080:8080 openwhisk/dockerskeleton
```

Comment:
should this be `dockerskeleton` or `whisk/dockerskeleton` when your working with local images?

Reply:
it could be just `dockerskeleton` too which would avoid the prefix/tag.